packages/cli/src/commands/certs/issue.ts

```diff
@@ -199,7 +199,7 @@ async function runStartOrder(
   const [header, ...rows] = dnsTable(
     pendingChallenges.map(challenge => {
       const subdomain = getSubdomain(challenge.domain);
-      if (!subdomain) {
+      if (subdomain === null) {
```

Comment:
What is the value here that led to the regression? `false` or `undefined`?

Reply:
Subdomain would be an empty string, and `!""` is `true`